hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -443,6 +447,15 @@ protected Ratio getRatio() {
     this.nsLookups = registry.counter(name(this.getClass(), NS_LOOKUPS, scope));
     this.nsLookupsFailed = registry.counter(name(this.getClass(), NS_LOOKUPS_FAILED, scope));
 
+    this.userRegionLockTimeoutCount =
+      registry.counter(name(this.getClass(), "userRegionLockTimeoutCount", scope));
+    this.userRegionLockWaitingTimer =
+      registry.timer(name(this.getClass(), "userRegionLockWaitingTimer", scope));
+    this.userRegionLockHeldTimer =
+      registry.timer(name(this.getClass(), "userRegionLockHeldTimer", scope));
```

Comment:
nit: to match the names of the other timers, perhaps
```suggestion
      registry.timer(name(this.getClass(), "userRegionLockWaitDurationMs", scope));
    this.userRegionLockHeldTimer =
      registry.timer(name(this.getClass(), "userRegionLockHeldDurationMs", scope));
```

(Although, I'm confused - is there anything that guarantees we get Milliseconds when we read out the metric? Or is it in nanoseconds? Other timer names in this file say `Ms` but now I have my doubts. 🤔 )

Reply:
We don't have a universal convention here but in my opinion the default time unit is millisecond so we should not put the time unit into the metric name if the metric's unit is milliseconds.